examples/nlp/token_classification/data/prepare_data_for_punctuation_capitalization.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+The script converts raw text to the NeMo format for punctuation and capitalization task.
```

Comment:
Can we have more detail on raw text format and nemo format? with an example command.

Reply:
done